cli/flags/options.go

```diff
@@ -23,17 +23,9 @@ const (
 	// FlagTLSVerify is the flag name for the TLS verification option
 	FlagTLSVerify = "tlsverify"
 	// FormatHelp describes the --format flag behavior for list commands
-	FormatHelp = `Format output using a custom template:
-'table':            Print output in table format with column headers (default)
-'table TEMPLATE':   Print output in table format using the given Go template
-'json':             Print in JSON format
-'TEMPLATE':         Print output using the given Go template.
-Refer to https://docs.docker.com/go/formatting/ for more information about formatting output with templates`
+	FormatHelp = `Format output using a custom template ("table|table TEMPLATE|json|TEMPLATE"). Refer to [formatting page](https://docs.docker.com/go/formatting/) for more information.`
 	// InspectFormatHelp describes the --format flag behavior for inspect commands
-	InspectFormatHelp = `Format output using a custom template:
-'json':             Print in JSON format
-'TEMPLATE':         Print output using the given Go template.
-Refer to https://docs.docker.com/go/formatting/ for more information about formatting output with templates`
+	InspectFormatHelp = `Format output using a custom template ("json|TEMPLATE"). Refer to [formatting page](https://docs.docker.com/go/formatting/) for more information.`
```

Comment:
Hm... I wonder if we can fix this in some other way. I know links in the output tend to be error-prone (users copy the link, and formatting code with it), but also to provide at least the "basic" information in the output.

Could we adjust the generator to replace newlines with `<br />` ?

Reply:
> Could we adjust the generator to replace newlines with `<br />` ?

Sounds good :+1:

Reply:
Opened https://github.com/docker/cli-docs-tool/pull/33 to fix this.